antarest/study/business/optimization_management.py

```diff
@@ -136,62 +106,11 @@ class OptimizationFormFields(FormFieldsBaseModel):
     },
     "unfeasible_problem_behavior": {
         "path": f"{OPTIMIZATION_PATH}/include-unfeasible-problem-behavior",
-        "default_value": UnfeasibleProblemBehavior.ERROR_VERBOSE,
+        "default_value": UnfeasibleProblemBehavior.ERROR_VERBOSE.value,
```

Comment:
here you can directly use the enum member. When the enum member is serialized in Pydantic, it will automatically use its value

Reply:
Not changed